og.permissions.yml

```
@@ -1,4 +1,4 @@
-administer group:
+administer organic groups:
   title: 'Administer Organic groups permissions'
```

This permission allows much more than managing permissions, no?
```suggestion
  title: 'Administer Organic groups'
```